tailwind.config.js

```diff
@@ -1,11 +1,35 @@
+const colors = require('tailwindcss/colors')
+const path = require('path');
+const distPath = path.resolve(__dirname, 'src');
+
 module.exports = {
   purge: [],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    borderColor: theme => ({
+      ...theme('colors'),
+      'rust': "#e56034"
+    }),
+    minWidth: {
+      'project-card-mobile': "320px",
+      'project-card': "420px"
+    },
```

Comment:
Doesn't this belongs to the CSS itself? We have media queries and also the `@screen` directive.
Check this out: https://tailwindcss.com/docs/breakpoints and https://tailwindcss.com/docs/functions-and-directives#screen